src/app/admin/admin.component.html

```diff
@@ -143,7 +143,7 @@
         >
           Graylist&nbsp;
         </button>
-        <button *ngIf="submittingGraylistUpdate" class="btn btn-dark fs-15px ml-5px" disabled>Working...</button>
+        <button *ngIf="submittingGraylistUpdate" class="btn btn-warning fs-15px ml-5px" disabled>Working...</button>
```

Comment:
warning shouldn't be used here. makes it a yellow color, should match whatever color we have in the non-disabled state.